ana_scans.py

```diff
@@ -0,0 +1,227 @@
+#!/bin/env python
+
+def launchAna(args):
+  return launchAnaArgs(*args)
+
+def launchAnaArgs(anaType, cName, cType, scandate1, scandate2=None, ztrim=4.0, chConfigKnown=False, channels=False, panasonic=False):
+  import os
+  import subprocess
+  from subprocess import CalledProcessError
+  from anaInfo import ana_config
+  from gempython.utils.wrappers import runCommand
+
+  dataPath  = os.getenv('DATA_PATH')
+  dirPath   = ""
+  elogPath  = "%s/%s"%(os.getenv('ELOG_PATH'),scandate1)
+    
+  print "Analysis Requested: %s"%(anaType)
+
+  #Build Commands
+  cmd = ["python"]
```

Comment:
Great okay, I'll work on a setup script for vfatqc-python-scripts...I have a prototype already.